docs/configure/policy/rules.md

```diff
@@ -121,6 +121,45 @@ if (ActionItem.Namespace === "api") {
 }
 ```
 
+### PagerDuty Incidents
+If you have attached a PagerDuty installation to your organization, you can use the
+`createPagerDutyIncident` function to create incidents. The function takes two arguments:
+
+* from - the email address of a valid user on the PagerDuty account
+* incident - an object that expects the following properties:
+  * title - a summary of the incident
+  * serviceID - the id of the service that the incident belongs to
+  * urgency - the urgency of the incident. Valid values are `high` or `low`
+  * bodyDetails (optional) - provides a detailed description of the incident
+  * escalationPolicyID (optional) - assign the incident to an escalation policy instead of assigning directly to a user
+  * assignmentIDs (optional) - a list of user IDs (only one assignee is supported at this time) to assign to the incident. Cannot be provided if escalationPolicyID is already specified.
+
+#### Examples
+```js
+if (ActionItem.Severity >= CRITICAL_SEVERITY && ActionItem.IsNew) {
+  incident = {
+		"title": ActionItem.Title,
+		"serviceID": "PIIWGG1",
+		"urgency": "high",
+		"bodyDetails": ActionItem.EventType
+	}
+	createPagerDutyIncident("insights@acme-co.com", incident)
+}
+```
+
+```js
+if (ActionItem.Severity >= CRITICAL_SEVERITY && ActionItem.IsNew) {
+  incident = {
+		"title": "Fixed title!",
+		"serviceID": "PIIWGG1",
+		"urgency": "high",
+		"bodyDetails": "Fixed body details",
+		"assignmentIDs": ["P6GC8ZZ"]
+	} 
+	createPagerDutyIncident("insights@acme-co.com", incident)
+}
```

Comment:
not sure what this example is showing - anything that isn't covered by the first example?

Reply:
🤔 Maybe we can combine the two? WDYT

Reply:
We could, my idea was to have an example showing that you don't **need** to have an assignment for it (`assignmentIDs` or `escalationPolicyID`)

Reply:
I combine them and set a JS comment to let users know it is _optional_